ui/src/components/sermons/RelatedSermonCard.js

```diff
@@ -1,23 +1,32 @@
-import { AspectRatio, Box, Image, Text, Stack, VStack } from "@chakra-ui/react";
-import { Link } from "react-router-dom";
-import { DateTime } from "luxon";
+import { AspectRatio, Box, Image, Text, Stack, VStack } from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
 
 const RelatedSermonCard = ({ sermonData, allSermons }) => {
   let sermonDate = DateTime.fromISO(sermonData.datePreached).toFormat(
-    "LLLL dd, yyyy"
+    'LLLL dd, yyyy'
   );
-  let sermonImage = "";
+  let sermonImage = '';
   if (sermonData.sermonSeries[0].image !== null)
     sermonImage = sermonData.sermonSeries[0].image.sourceUrl;
-  else sermonImage = process.env.PUBLIC_URL + "/images/ripple_black.svg";
+  else sermonImage = process.env.PUBLIC_URL + '/images/ripple_black.svg';
+
+  const [onlineSermon, setOnlineSermon] = useState(false);
```

Comment:
line 15-21 is unused. please make it a habit to check for these things before pushing and asking for review. thanks